stream2hop/tns2hop.py

```diff
@@ -0,0 +1,254 @@
+import argparse
+from . import Utils as ut
+from twisted.internet import task, reactor
+from datetime import datetime, timedelta
+import requests
+import json
+import wget
+import csv
+import os
+
+
+def _add_parser_args(parser):
+    """
+        adding parser arguments
+    """
+    ut.add_common_arguments(parser)
+    parser.add_argument("--api_key", help="TNS Bot's api key")
+
+
+def search(url, json_list, api_key):
+    """
+      Search for objects in TNS
+
+      Args:
+        url: TNS url
+        json_list: parameters in the search query
+        api_key: TNS bot's api key
+
+      Return:
+        search result
+    """
+    try:
+        # url for search obj
+        search_url = url + "/search"
+        # construct the list of (key,value) pairs
+        search_data = [
+            ("api_key", (None, api_key)),
+            ("data", (None, json.dumps(json_list))),
+        ]
+        # search obj using request module
+        response = requests.post(search_url, files=search_data)
+        return response
+    except Exception as e:
+        return [None, "Error message : \n" + str(e)]
+
+
+def get_object(url, json_list, api_key):
+    """
+      Get TNS object details
+
+      Args:
+        url: TNS URL
+        json_list: parameters passed to TNS get API specifing the information needed
+        api_key: TNS bot's api key
+
+      Returns:
+        Object's data
+    """
+    try:
+        get_url = url + "/object"
+        get_data = [
+            ("api_key", (None, api_key)),
+            ("data", (None, json.dumps(json_list))),
+        ]
+        response = requests.post(get_url, files=get_data)
+        return response
+    except Exception as e:
+        return [None, "Error message : \n" + str(e)]
+
+
+def fix_photometery(object, parameters_list):
```

Comment:
May be worth renaming the function `fix_photometry()`

Reply:
Oops didn't see that!,, will change it